### PR TITLE
Use Options rather than transients for state

### PR DESCRIPTION
**Pull request by joehoyle**
Transients will be put in the Object Cache if it's persistent

### CODE REVIEW

File: plugin.php [hunk 1]
```diff
@@ -40,7 +40,7 @@ function get_broker_id() {
 	 *
 	 * @param string $id Broker ID. Defaults to home URL.
 	 */
-	return apply_filters( 'authbroker.id', home_url() );
+	return apply_filters( 'authbroker.id', home_url( '/' ) );
```
Comment (rmccue) [inline plugin.php:43]: This should be in a separate PR :)

Comment (joehoyle) [inline plugin.php:43]: :shipit:
